app/uk/gov/hmrc/ngrloginregisterfrontend/controllers/ConfirmUTRController.scala

```diff
@@ -90,7 +91,7 @@ class ConfirmUTRController @Inject()(view: ConfirmUTRView,
   }
 
   def submit(): Action[AnyContent] =
-    authenticate.authWithUserDetails.async { implicit request =>
+    (isRegisteredCheck andThen authenticate).async { implicit request =>
```

Comment:
I wonder if we need an action out there that covers both so it can be called with a single name

Reply:
I've kept them separate to not over complicate the actions, and make them easy to maintain and read we can easily add more actions as well this way if we have any changes